copylot/assemblies/photom/utils/pattern_tracing.py

```diff
@@ -0,0 +1,59 @@
+from typing import List, Tuple 
+from PyQt5.QtCore import Qt, QPoint
+from PyQt5.QtGui import QPolygon
+import logging 
+
+class Shape(QPolygon):
```

Comment:
let's rename this to something more meaningful as well. trace_shape or similar.

Reply:
you mean renaming the shape class -> trace_shape?

Reply:
@aaronalvarezcz address this. Let's not name it shape because that is too common of a name and is used in a bunch of other packages. Name it  consistent to what you have named the other functions. (i.e Shape_tracer, shape_drawer,etc)

Reply:
still pending@aaronalvarezcz